### DISCUSSION TITLE
images not loading on website

### DISCUSSION THREAD

**Discussion by alfa121** [community/community#48051] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I can see the images of my website in my local server but this is not the case in github pages.
I extensively looked into the most common problems such as: capitalized image extensions, path issues, and I still can not figure it out.

Here are the repository and website:  

https://github.com/alfa121/codecademy-Tea_Cozy

https://alfa121.github.io/codecademy-Tea_Cozy/

**Comment by airtower-luna** [comment, marked as answer]
There's a path problem here: https://github.com/alfa121/codecademy-Tea_Cozy/blob/9d70705f0b8d08047e051ad1fc301500cbea3221/index.html#L14

That link is using an absolute path (starting with a slash), so the browser will look at `https://alfa121.github.io/images/img-tea-cozy-logo.png`, which doesn't exist. Make it a relative path and it'll work:

```html
        <img class="logo" src="images/img-tea-cozy-logo.png" alt="tea cozy logo">
```

**Reply by alfa121** [reply]
I had tried this option but the browser would not make the changes available.  I made the changes again and opened as incognito tab and it worked. Is it the cache the culprit for this?
Thank you for your help!

**Reply by airtower-luna** [reply]
Could be the browser cache, could be that the site needs a few minutes to deploy, possibly a bit of both. :sweat_smile: 

If you have a similar issue you can try to reload without browser cache (in Firefox that's Ctrl+Shift+R, not sure about other browsers), so at least you won't get an old version from the browser cache. :wink:

**Comment by ihnatenkoo** [comment]
Hello, wrong image path
**Correct path is:**
![i1](https://user-images.githubusercontent.com/89586345/220156468-44edb1f9-799e-44e3-9b19-6fc6a686c264.png)
But you path is:
![i2](https://user-images.githubusercontent.com/89586345/220156519-10e874da-2c11-49d9-9ae4-bcd572872edb.png)
